src/rdb.c

```diff
@@ -3765,6 +3765,12 @@ void killRDBChild(void) {
      * This includes:
      * - resetChildState
      * - rdbRemoveTempFile */
+
+    /* However, there's a chance the child already exited, and will not recieve the signal,
```

Comment:
the comments above seem outdated.

Reply:
i'm arguing that it's still valid, that's why i start the new one with "However, ".
since we don't do `waitpid`, in most cases we **won't** take these actions (e.g. resetChildState).
only in rare cases we'll detect the child already exited, and then we'll do these things.